Database/Corrections/cataQuestFixes.lua

```diff
@@ -3116,6 +3116,7 @@ function CataQuestFixes.Load()
         },
         [24707] = { -- The Ballad of Maximillian
             [questKeys.preQuestSingle] = {24706},
+            [questKeys.objectives] = {{{38708,nil,Questie.ICON_TYPE_EVENT}}}
```

Comment:
Why do we need an event icon for this? The quest seems to ask the player to kill [the queen](https://www.wowhead.com/cata/npc=38708/devilsaur-queen)

Reply:
@BreakBB good point - I have changed to an extraObjective to start the event https://github.com/Questie/Questie/pull/6658/commits/3dbf1990bb206d03726082e8dd10015ac5924668

Reply:
Oh, Maximillian is a naughty NPC. Did you fix coords for it for every location?

Reply:
He has a static point but can also follow you around - how would you fix coords for that?

Reply:
i'd just add spawns to each spot where he's supposed to be near you for the said quests. i guess we need some phase spawns for this, i'll do when i get to that zone

Reply:
@BreakBB going to resolve this for now - phasing idea is sensible but I don't know how to do it, and it's not a blocker for any of these other changes